Sharabesh2

Sharabesh opened the pull request:
This includes the changes suggested in the email.

tobinsarah commented:
new file:   .DS_Store
	new file:   .SR_Board.brd.lck
	new file:   .SR_Board.sch.lck
	new file:   Board.brd
	new file:   Board.s#1
	new file:   Board.s#2
	new file:   Board.s#3
	new file:   Board.s#4
	new file:   Board.sch

Sharabesh commented:
I think I went through and removed all the extra files. I also updated my gitignore, but I'm not sure if that fixed the problem. Let me know if anything else needs to be removed.

tobinsarah commented:
Closing pull request--- may end up removing last commit if it is redudant with history from RevA
